extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-mysql/src/main/resources/guac-manifest.json

```diff
@@ -24,7 +24,8 @@
         "translations/es.json",
         "translations/fr.json",
         "translations/ja.json",
-        "translations/ru.json"
+        "translations/ru.json",
+        "translations/zh.json"
```

Comment:
Shouldn't this also be added to the `guac-manifest.json` file for PostgreSQL and SQL Server?

Reply:
okay，I add。